sdc/tests/test_dataframe.py

```diff
@@ -1143,15 +1143,46 @@ def test_impl(A):
         hpat_func = self.jit(test_impl)
         pd.testing.assert_frame_equal(hpat_func(df), test_impl(df2))
 
-    @skip_numba_jit
-    def test_df_reset_index1(self):
+    def test_df_reset_index_drop_True(self):
         def test_impl(df):
             return df.reset_index(drop=True)
```

Comment:
These tests are OK, but both of them have drop=True/False hardcoded in the jitted function body, so you only test that your implementation works with literal bool param. To test that it works with variable of type Boolean you need to define a variable in test and then pass this var into the jitted function, e.g:
```
            for skipna_var in [True, False]:
                actual = hpat_func(S, skipna=skipna_var)
                expected = test_impl(S, skipna=skipna_var)
```
And such test is definitely needed.